amimals/Hunting.java

```diff
@@ -0,0 +1,5 @@
+package amimals;
+
+public interface Hunting {
```

Comment:
Possibly, HuntingAnimal should be better, because an object of its type will be an animal, not an action. But also possible. Play has the same thing